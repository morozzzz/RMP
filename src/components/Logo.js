import React from 'react';
import './Logo.css';

const Logo = () => (
    <a href="./" className="logo">netflixroulettee</a>
);

export default Logo;
