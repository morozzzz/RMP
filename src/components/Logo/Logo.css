.logo {
    text-decoration: none;
    color: rgb(248, 129, 129);
}