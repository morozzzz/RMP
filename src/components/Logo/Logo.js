import React from 'react';
import './Logo.css';

const Logo = ({ name, redirectUrl }) => (
    <a href={redirectUrl} className="logo">{name}</a>
);

export default Logo;
