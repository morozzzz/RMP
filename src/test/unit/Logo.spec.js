import React from 'react';
import { shallow } from 'enzyme';
import Logo from '../../components/Logo/Logo';

describe.skip('Logo', () => {
    it('should be rendered correctly', () => {
        const logoName = 'some name';
        const logoRedirectUrl = 'http://some_url';
        const component = shallow(<Logo name={logoName} redirectUrl={logoRedirectUrl} />);

        expect(component).toMatchSnapshot();
    });
});
