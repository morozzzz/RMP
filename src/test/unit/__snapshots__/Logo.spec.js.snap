// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should be rendered correctly 1`] = `
<a
  className="logo"
  href="http://some_url"
>
  some name
</a>
`;
